pkg/kubectl/resource/builder.go

```diff
@@ -60,6 +60,7 @@ type Builder struct {
 
 	singleResourceType bool
 	continueOnError    bool
+	skipFetchingNames  bool
```

Comment:
I prefer that we name things not in the negative, makes code easier to read.

So make this fetchNames